feat(content): New "obsolete" ships: Monitor & Jackal + Heavy Blaster (#11358)
endless-sky/endless-sky: data/human/ships.txt

ship "Jackal"
	sprite "ship/jackal"
	thumbnail "thumbnail/jackal"
	attributes
		category "Interceptor"
		"cost" 320000
		"shields" 1600
		"hull" 500
		"required crew" 1
		"bunks" 2
		"mass" 100
		"drag" .95
		"heat dissipation" .68

> 		"fuel capacity" 300

Reviewer: ```suggestion
		"fuel capacity" 500
```

		"cargo space" 10
		"outfit space" 155
		"weapon capacity" 25
		"engine capacity" 40
		weapon
			"blast radius" 21
			"shield damage" 210
			"hull damage" 105
			"hit force" 315
	outfits
		"Modified Blaster" 4
		
		"nGVF-BB Fuel Cell"
		"LP036a Battery Pack"
		"D14-RN Shield Generator"
		
		"Chipmunk Plasma Thruster"
		"Chipmunk Plasma Steering"
		"Hyperdrive"

	engine -10 35
	engine 10 35
	gun -17 -8.5
	gun 17 -8.5
	gun -25 -6
	gun 25 -6
	leak "leak" 50 40
	leak "flame" 70 80
	explode "tiny explosion" 10
	explode "small explosion" 10
	"final explode" "final explosion small"

> 	description "The Jackal is notable for being one of the earliest warships that was designed in mind for "freelancer" pilots, rather than for corporate or law enforcement usage. In its heyday, the Jackal contended with the Berserker for the title of the most popular Interceptor-class ship in human space. However, the Jackal's age and inflexibility with regards to new developments in weaponry led to a decline in sales that eventually bankrupted its manufacturer, ending a century-long production run - a phenomenon its age-old rival, the Berserker, may only have avoided thanks to Betelgeuse Shipyards' other, more popular models."

Reviewer: ```suggestion
	description `The Jackal is notable for being one of the earliest warships that was designed in mind for "freelancer" pilots, rather than for corporate or law enforcement usage. In its heyday, the Jackal contended with the Berserker for the title of the most popular Interceptor-class ship in human space. However, the Jackal's age and inflexibility with regards to new developments in weaponry led to a decline in sales that eventually bankrupted its manufacturer, ending a century-long production run - a phenomenon its age-old rival, the Berserker, may only have avoided thanks to Betelgeuse Shipyards' other, more popular models.`
```

> 	description `The Jackal is notable for being one of the earliest warships that was designed in mind for "freelancer" pilots, rather than for corporate or law enforcement usage. In its heyday, the Jackal contended with the Berserker for the title of the most popular Interceptor-class ship in human space. However, the Jackal's age and inflexibility with regards to new developments in weaponry led to a decline in sales that eventually bankrupted its manufacturer, ending a century-long production run - a phenomenon its age-old rival, the Berserker, may only have avoided thanks to Betelgeuse Shipyards' other, more popular models.`

Reviewer 2: The third sentence is very long and has a lot of hyphens and dashes... I wonder if it would be possible to split it up into two sentences?

ship "Jackal"
	sprite "ship/jackal"
	thumbnail "thumbnail/jackal"
	attributes
		category "Interceptor"
		"cost" 320000
		"shields" 1600
		"hull" 500
		"required crew" 1
		"bunks" 2
		"mass" 100
		"drag" .95
		"heat dissipation" .68
		"fuel capacity" 300
		"cargo space" 10
		"outfit space" 155
		"weapon capacity" 25
		"engine capacity" 40
		weapon
			"blast radius" 21
			"shield damage" 210
			"hull damage" 105
			"hit force" 315
	outfits
		"Modified Blaster" 4
		
		"nGVF-BB Fuel Cell"
		"LP036a Battery Pack"
		"D14-RN Shield Generator"
		
		"Chipmunk Plasma Thruster"
		"Chipmunk Plasma Steering"
		"Hyperdrive"

	engine -10 35
	engine 10 35
	gun -17 -8.5
	gun 17 -8.5
	gun -25 -6
	gun 25 -6
	leak "leak" 50 40
	leak "flame" 70 80
	explode "tiny explosion" 10
	explode "small explosion" 10
	"final explode" "final explosion small"

> 	description "Jackal."

Reviewer 3: Well, I had "The Jackal was the first small ship humans made. Up to this point, traveling between planets required civilians to book passages on slow spacecraft, who would be attacked by similar lumbering giants controlled by pirates. When the Jackal was created, all of that changed. It was suddenly possible for a civilian to reach any planet in a matter of days with relatively small danger.", and then realized the Jackal only has two bunks. So... how about
```suggestion
	description "One of the first small combat ships humanity built, the Jackal saw most of its use fighting anarchists along the Rim... or being used as a pirate interceptor to attack civilians in the North."
```

Author: I don't think the Jackal would have been one of the first small combat ships. This might be an older interceptor, but it's not like the Warner Bro's first plane of humanity's interceptors. It may have been, say, the first truly mainstreamed interceptor back in its day, but it certainly would not be one of the first combat ships of humanity.

Reviewer 4: ```suggestion
	description "The Jackal is notable for being the earliest extant warship that was designed in mind for "freelancer" pilots, rather than for corporate or law enforcement usage. In its heyday, the Jackal contended with the Berserker for the title of the most popular Interceptor-class ships in human space. However, the Jackal's age and inflexibility for new developments in weaponry has all but relegated the Jackal to the scrapheap - a phenomenon its age-old rival, the Berserker, is now facing today."
```
How does this sound? It's adapted from an earlier draft; puts it around 2800 on the timeline.

Reviewer 5: I wouldn't necessarily state that it's been relegated to the scrapheap, otherwise everyone in human space would've already tossed theirs in the scrapyard.  Too negative a connotation.  It's a pretty ship, and I think there may be another avenue.

I really liked the general lore idea from when this was still the Recruiter in the Rumskib mod, except let's say the lore there is reversed - the idea instead is that it is a vintage Lionheart construction, obsolete now but serving beside the Scout as a support ship when it was a Freelancer, and the Scout was new-ish.  Imagine this ship being the Headhunter's direct predecessor.

In that regard, it's not a bad ship, but it's not up to modern standards and is vestigial of lighter combat-capable ships that are simply too big to be interceptors and too small to be light warships.  That no man's land of master-of-none.

Lore-wise, it wouldn't even need a name change.  Lionheart has mammalian names.  Jackals are around Lions.  Oh man, I actually quite love it.

Reviewer: I suppose instead of `has all but relegated the Jackal to the scrapheap - a phenomenon its age-old rival, the Berserker, is now facing today`, you could go with something like `led to a decline in sales that eventually bankrupted its manufacturer, ending a century-long production run - a phenomenon its age-old rival, the Berserker, may only have avoided thanks to Betelgeuse Shipyards' other, more popular models.`

Author: @bene-dictator Any preference between Doom's suggestion to/with your pitch, or Doyle's pitch? I'm leaning towards the former as I imagine the Jackal as less-so an exploration ship and more-so specifically referencing its interceptor role, and I do like the reference to the Berserker.

If we're considering the latter, Doyle, it seems like you're pushing the Jackal less into a combat role and more into a sort of old multi-purpose explorer ship that also happens to be an interceptor, while the initial suggestion was more for that combat role. Yes, 300 fuel is pretty low, but 3 jumps is plenty for most of human space. A *ton* of ships sit in that 500 jump range, so it feels a bit bland.

It seems proper that an old human interceptor has less fuel capacity than the popular Shuttle, that, shuttles, when the role of the Jackal is more for combat. (Also, if you look at other old ships, they too have pretty underwhelming stats in some areas; I feel like trying to make everything "decent" or "meh" on these ships defeats the purpose, and actually having something "comparably bad" on them is more their flavor.

Reviewer 5: I would like to clarify the reasoning I used; redeeming characteristics of survivability and "why" humans would still be using it.  In regard to interceptors, 500 fuel would make this the highest capacity non-Marauder interceptor in human space, which would be incentive for players geared toward early game courier work to actually use it.  It can also keep up as part of a fleet in transit.  Say we scrub the "expand cargo" aspect and simply make it a long-range interceptor - for modern regions it's not as justifiable to produce anymore, but for anyone without the infrastructural support, it is a good investment.

If you're a pirate or anarchist with 300 fuel, jumping from Almaaz to Rigel to raid Republic shipping, you're out of fuel as soon as you get there (low chances of escape) - 500 fuel lets you get there and jump back to Mintaka.  You can also fully circle Almaaz > Mintaka > Tortor > Gorvi > Mintaka > Almaaz for a complete patrol route ending at home.  Find something nice to pillage?  Call the Monitor, they take a direct route w/o wasted time.  300 makes this a marginally stronger Sparrow.  Sparrow's more of a modern convenience; you have plenty of rest stops for most paths you take.  Jackal's for when you don't have those rest stops as often, as human space used to be when it was designed.

So it's not to envision a multi-purpose explorer ship, but rather a long range interceptor that is able to run support missions for stranded vessels and remote courier work should the contract require it.

Author: I think you're missing a bit of the point of these ships - the idea isn't solely that they're meant to have reasons they're "still in use" by choice, at least, not for a majority of them, nor is it to incentivize players or marauders to use it. The concept more is to show that there are indeed ships from humanity's past that are still floating around, not in shipyards anymore, but are in the hands of anarchists for various reasons.

To put it another way, these are old cars that are still rarely seen nowadays that have poor mileage or usage compared to today's cards, but have aged and are still present due to reasons such as a lack of funding to purchase anything else, long-time ownership resulting from attachment, or perhaps hand-me-downs. It being a marginally stronger sparrow *is good*, it shouldn't be much better than one, as it's much older, and the Sparrow should have reason why it's sold and this no longer is.

These aren't meant to be competitive or have an efficient operating span within what's *current* human space - the idea more is to consider what historically may have been the case, also considering that the Jackal was a common ship before. In older times, presumably humans hadn't explored out to far reaches, or at least, their ships may not have been designed to, unless specifically for those roles, such as the Scout.

In the case of the Jackal, it's meant to be, *of it's time*, a fast interceptor with four gun ports. So in such a case, the combat is the focus, having a small, dogfighter ship earlier on in history. Instead of these being the farther-ranged vessels, it might just be the case that larger ones like the Monitor, or other ships that may have been at the time, carried more fuel for further operations.

Author: I will ask @Quantumshark his opinion as well though, since I worked with him together on the stats to differentiate the efficiency of these ships from modern ones.

Reviewer 5: That's a very fair point; I should be looking at it as living lore, rather than something to practically consider for a playthrough.  As someone that likes Classic cars, there is an "age-limit" where they become more and more unworthy of the modern road, and I can do better to apply that same logic here.

I still think it would be an interesting fit as an early ship and predecessor to the Headhunter that put Lionheart on the map as a major manufacturer beyond City-Ships, and I would like to voice encouragement for +10 cargo for the total of 20 so an outfit expansion module could potentially allow for more loadout variation.

ship "Monitor"
	sprite "ship/monitor"
	thumbnail "thumbnail/monitor"
	attributes
		category "Medium Warship"
		"cost" 2900000
		"shields" 4900
		"hull" 2400
		"required crew" 20
		"bunks" 32
		"mass" 690
		"drag" 8.4
		"heat dissipation" .32

> 		"fuel capacity" 300

Reviewer: ```suggestion
		"fuel capacity" 500
```

		"cargo space" 20
		"outfit space" 330
		"weapon capacity" 130
		"engine capacity" 90
		weapon
			"blast radius" 73
			"shield damage" 730
			"hull damage" 365
			"hit force" 1095
	outfits
		"Heavy Blaster" 3
		"Laser Turret" 2
		"Anti-Missile Turret"
		
		"KP-6 Photovoltaic Array" 3
		"nGVF-BB Fuel Cell"
		"LP036a Battery Pack"
		"D23-QP Shield Generator"
		"Laser Rifle" 6
		"Fragmentation Grenades" 4
		
		"X3700 Ion Thruster"
		"X3200 Ion Steering"
		"Hyperdrive"
		
	engine -20.5 92
	engine 20.5 92
	gun 0 -94.5 "Heavy Blaster"
	gun -11 -86 "Heavy Blaster"
	gun 11 -86 "Heavy Blaster"
	turret -17.5 -19.5 "Laser Turret"
	turret 17.5 -19.5 "Laser Turret"
	turret 0 17 "Anti-Missile Turret"
	leak "leak" 30 40
	leak "flame" 40 80
	leak "big leak" 70 30
	explode "tiny explosion" 20
	explode "small explosion" 40
	explode "medium explosion" 20
	explode "large explosion" 5
	"final explode" "final explosion medium"

> 	description `When the Republic first formed, planetary governments and corporations often fielded their own fleets; only the Deep had any form of a combined navy. Earth often found itself inefficiently directing those fleets to fight against pirates or defiant worlds - with a need to bring "peace among the stars," per the Republic motto, the newly formed Republic Navy commissioned the shipyards on Luna to design a robust warship that would be able to project the power of a united Republic. The resulting ship, the Monitor, served as the mainstay of the Navy fleets until the introduction of the more advanced Frigate.`

Reviewer 2: ```suggestion
	description `When the Republic first formed, planetary governments and corporations often fielded their own fleets; only the Deep had any form of a combined navy. Earth often found itself inefficiently directing those fleets to fight against pirates or defiant worlds. With a need to bring "peace among the stars," per the Republic motto, the newly formed Republic Navy commissioned the shipyards on Luna to design a robust warship that would be able to project the power of a united Republic. The resulting ship, the Monitor, served as the mainstay of the Navy fleets until the introduction of the more advanced Frigate.`
```

> 	description "Before the time of the Navy, the formation of the Republic caused civil unrest amongst the inhabitants of several worlds. While some corporations happily joined the fledgling power, others wished to stay independent or even rebelled against governance. To both quell this uncertainty and bring order to the growing Republic, the Monitor was manufactured as a powerful security ship, armed with three large forward guns and three supportive turret mounts. These ships would also see use in the Alpha Wars, although they were eventually made obsolete by more robust designs such as the Firebird."
> 	description "	In a twisted act of irony, aging Monitors have found themselves in the hands of those against whom they once fought, including groups of anarchists that now populate the north of human space."

Reviewer 4: The Alpha Wars predate the formation of the Republic by nearly 300 years. Do we want the Monitor to be as old as the Firebird? I feel like it would be more appropriate for it to be a newer design, maybe have it be a predecessor of the Frigate.

```suggestion
	description "In the early days of the Republic, planetary governments and corporations often fielded their own fleets; only the Deep had any form of public space navy. The Republic's rapid expansion caused civil unrest on several worlds, and Earth often found itself directing the fleets of loyal planets against rebellious ones. With a need to quell the growing uncertainty and bring order across human space, the newly formed Navy commissioned the shipyards on Luna to design an advanced warship that would be able to project the power of a united Republic. the ship they produced, the Monitor, made up the backbone of the early Navy until the introduction of the more robust Frigate."
	description "	Ironically, the few aging Monitors that survive today have found themselves in the hands of those against whom they once fought, including groups of anarchists that now populate the north of human space."
```

The idea is that the shipyards on Luna were the first "Republic Navy Yard" until the Paradise Worlds joined, and then they began to dominate the Republic with their shipyards on Geminus.

Author: I was considering the Monitor as older than the Firebird, yes. I don't think the Monitor would be appropriate as a newer design after something like the Firebird as the Firebird can be seen as superior in many ways, and would be the main front-line warship at that time/was the pretty well-known one.

Like the other ships such as the Enforcer, it would have been a much earlier-on combat ship, before there were many others or around the time that some only began to be constructed, as a sort of "Cruiser" that other ships couldn't easily compete with that secured or enforced sectors of human space.

Perhaps then it was before even the time of the Republic? The main draw I wanted to make was it was the "Frigate" or "Cruiser" or whathaveyou of older days before something big like the Navy exists. I'm not sure how well it could fit before the existence of both the Navy and Firebird though.

Author: They still could have been in the Alpha Wars, but could have already been aging at the time.

Reviewer 4: > I don't think the Monitor would be appropriate as a newer design after something like the Firebird as the Firebird can be seen as superior in many ways, and would be the main front-line warship at that time/was the pretty well-known one.

I don't see why the Monitor can't be both more recent and also inferior in some ways to the Firebird. It's a design that has been dropped in favour of a better one, the Frigate. I also feel like this description would be a good fit to describe the formation of the Navy and its early ship composition, and the role of Earth in the early Republic.

> Like the other ships such as the Enforcer, it would have been a much earlier-on combat ship, before there were many others or around the time that some only began to be constructed, as a sort of "Cruiser" that other ships couldn't easily compete with that secured or enforced sectors of human space.

Is the Enforcer really as old as the Firebird either? I feel like we shouldn't have too many extant ships from before 2600, and I imagine the Enforcer probably would have been around after 2600, but not before that.

(2600: The Deep and the Paradise Planets have grown into powerful, isolated enclaves of high technology. The inhabited Core worlds and much of the Rim have fragmented into individual systems or small regions doing constant battle with each other and funding privateers to steal cargo from the others.)

And we also have to consider the place of the Leviathan, which has "been in service long before the formation of the galactic Republic." That fits the bill of being the old "Cruiser" more than any other ship, I feel.

> They still could have been in the Alpha Wars, but could have already been aging at the time.

You must be really desparate to fly around in a ship that was already considered outdated five hundred years ago...

Reviewer 4: The bottom line for me is that I don't think it makes sense for the Monitor to be older than the Firebird and still be used, without impinging on the uniqueness of the Firebird also fulfilling that role. I think we should leave the Firebird as the oldest model of ship still available in human space.

I imagine the timeframe of the older human ships would look something like:

```
2400s: Firebird
- Alpha Wars begin in the 2450s
2500s: Aerie, Leviathan
- Caravan Era begins in the 2620s
2600s: Enforcer, Hogshead, Saber
- Republic forms in the 2730s
2700s: Monitor, Frigate, Hauler, Scout
- Space renaissance
2800s: Berserker, Jackal, Star Queen, Bactrian
```

Author: If that's the case, then I feel the Monitor looses a lot of lore value simply being another design amongst several shipyards versus its original concept of an early large security ship (aka, it was the most powerful of its time, not an option or one of several choices for such a ship). It's not meant to be a Navy ship or one the Navy replaces, it's meant to come before the Navy as a necessity for fending off Pirates or securing systems for the newly formed Republic before the Navy was formed, or if you would, "Monitor" core systems.

Perhaps it was the, or one of a few, ships that resulted in the formation of the Navy, so not part of that advanced ship project, but the earlier case or example of it. If it were a Navy-specific ship, which isn't what I had intended, it would need a whole different design. 

Yes, the Enforcer is older than the Firebird, unless I have my dates wrong. Both the Enforcer and Saber were added as ships during the Caravan era. Specifically, the Enforcer, and in this PR, the Monitor, would have been security force ships for their respective areas - Enforcers for corporations that required security against Pirate threats as humanity expanded, and Monitors as something like a more direct enforcement ship for core worlds as the Republic was formed (but before the Navy was fully established). If there's a way to keep it somewhat in that venue, that would be good, but I'd avoid it being specifically in the Navy after it was formed.

Reviewer 5: I like what Saugia's getting at, and I like what bene-dictator is getting at - maybe the Monitor was, for a time, like the Aerie - constantly revised and updated as time progressed from its inception in either the 2400s or 2500s.

I think it's passible that the Monitor could be as old as the Firebird, or at least close to it - perhaps it was the pre-Republic governments decision to have something that catered specifically to their needs on a turreted level than the Firebird offered?  The Monitor was it, until the budget increased by the establishment of the Republic.

If, after keeping up with upgrades as the Aerie did, were a candidate for the new Republic to serve as its warship of the time - the Frigate comes along and knocks it off its pedestal because the tax base of this new Republic can afford the superior technology, thus rendering the Monitor militarily obsolete in the 2700s but still useful and successful as a budget production security ship until like half a century later.  It's like any military weapons trial - something good comes along like the M1 Garand (Monitor), they do some improvements over time, then something better comes along like the M14 (Frigate).  It's a good excuse to end production while still having a few floating around in space.

Reviewer 4: Your dates are wrong, the Caravan Era was after the Alpha Wars.

> If that's the case, then I feel the Monitor looses a lot of lore value simply being another design amongst several shipyards versus its original concept of an early large security ship (aka, it was the most powerful of its time, not an option or one of several choices for such a ship).

But the existence of the Leviathan (and presumably any other heavy warship) makes it unlikely to be the most powerful of its time, _unless_ it is as old as the Firebird, and I would prefer it to not be that old. And although I suppose the description wasn't clear about this, I did intend it to be a design of the shipyard on Luna. Because there are a severe lack of ship models originating from the Sol system. I guess all Luna does is reproduce Betelgeuse ships...

> Monitors as something like a more direct enforcement ship for core worlds as the Republic was formed (but before the Navy was fully established). If there's a way to keep it somewhat in that venue, that would be good, but I'd avoid it being specifically in the Navy after it was formed.

Although the original description implied that the Monitor was developed as a ship for the Republic (but I guess not the Navy), I could work with this compromise. I still do feel it would be better as an early Navy ship.

Reviewer 5: > > But the existence of the Leviathan (and presumably any other heavy warship) makes it unlikely to be the most powerful of its time, _unless_ it is as old as the Firebird, and I would prefer it to not be that old. And although I suppose the description wasn't clear about this, I did intend it to be a design of the shipyard on Luna. Because there are a severe lack of ship models originating from the Sol system. I guess all Luna does is reproduce Betelgeuse ships...
> 
Could be that the Leviathan was a response to interfaction conflict involving fleets of Medium Warships heading up against each other?  Perhaps the CCOR had a contingent of them and Sabers, and Betelgeuse needed to make a Monitor-killer capable ship?

Reviewer 4: I like the idea, though by its description Sabers were produced at least in the 2620s (as it's a Caravan Era ship), and if the Republic was formed in 2730, then there's not much room for the Leviathan to have "been in service long before the Republic existed" if it existed some time after 2620.

On the other hand, I would not be opposed to expanding / rewriting the lore for the Leviathan... though we might start to be pushing the boundaries of the scope of this PR, I think it could be warranted 😆

Reviewer 4: > I considered it to be a Navy-specific ship, I would have designed it in the same way Gefu had done his Navy ships, or had him create one, meanwhile this was designed to be older than that.

I don't think an old Navy ship HAS to be in the style of the modern Navy at all, and I think it makes a lot of sense that designs would borrow from the successful Firebird. So I don't buy that.

It's like how cars thirty years ago are so much blockier than cars today, you wouldn't immediately know my car from 1998 is the same model of car as the most recent model, but they were made by the same company and are ostensibly the same model.

Author: If anything, I've underestimated how old the Firebird really is.

Author: >I don't think an old Navy ship HAS to be in the style of the modern Navy at all, and I think it makes a lot of sense that designs would borrow from the successful Firebird. So I don't buy that.

It's like how cars thirty years ago are so much blockier than cars today, you wouldn't immediately know my car from 1998 is the same model of car as the most recent model, but they were made by the same company and are ostensibly the same model.

I think it just comes down to a difference of opinion. I designed the ship as a much older ship, and you're not in favor of that and are looking for a different purpose. The thing with the car example is it also applies to what I'm going for - this could be one of the very old Jeeps you see occasionally around, while there are other still-but-less-old cars around like corvettes and the likes.

I can understand you're not in agreement with shoving it into another old role past the Firebird, though I hadn't designed it as a Navy-specific ship and am not really sure shoving it into the role because "it could fit" is a great reason to do so.

Author: After thinking it over, perhaps the Monitor was before some sort of Navy shipyard movement that lead to the current design theme in-game, and playing into that more, what if the Monitor was the young Navy's first mass-produced/successful warship? That would also explain why it still appears a bit today, if it had its own value.

Reviewer 5: I gotta chime in on the classic cars aspect; I have a vintage truck that's 55 years old.  What I can say is that you can lift the body and put it on a newer chassis; think a 1970 Ford truck body, on a 2014 Ford Chassis.  The point is - it is possible to keep the same exterior aesthetic, while vastly improving the functional aspects of the vehicle.  You could have modern springs, modern engine, modern brakes, modern electric, etc.  It was even done with a vintage truck around the era mine is; "1978 Ford F100 Eluminator Concept Vehicle" completely new chassis designed to handle two electric engines and a full set of Li-ion batteries.  Same classic style, completely new function.

So when factoring in the "age" of certain ships, a lot could have changed over the years internally.  Cut-off points can explain gaps in outfit quality / cargo space / shielding etc.  The Saber for instance is a very weak ship - I can't use it.  Sprite size fills about the same general void as the Monitor, and if I up the outfit space to 310 while modding, I can have it equipped with base weapon equivalents to the Monitor's full 330.  If the Saber lost "operating system support" before the Monitor, it totally explains why it's a weaker ship.  Likewise, the Monitor losing support around the time the Frigate & Navy happened, explains why it's a weaker ship as well.

It could've started on Luna, been improved by Betelgeuse engineers.  Lots of ways to make both angles work in really fulfilling ways that give the ship a unique heart.

Reviewer 4: > what if the Monitor was the young Navy's first mass-produced/successful warship

I think there's good reason for the Navy to have made their own early warship and then phase it out later, despite the existence of the Firebird; given how far away Betelgeuse is from Earth, and that Betelgeuse might not have been part of the early Republic (or have even been enemies), it makes a lot of sense to me. And I imagine the Republic wanted to support local industry instead of importing from elsewhere, it seems like the timeline of human space in ES is driven by trade protectionism for some reason haha

Author: Sure, that sounds good then. With these older ships, now even the Navy gets a little bit of older love. Well, in a sense at least - considering it's more love to the Anarchists in this manner, lol.

The only thing I'd want to be careful of is the stats, if the Monitor is meant to be a Navy ship, it probably shouldn't be a *poor* comparison to the Firebird, but considering the comparison between its build and the Heavy Blaster variant Firebird, they're not all that far off from one another (3 HB/ 2 Laser Turrets/AM, to 4 HB/1 QB Turret, 1AM).

I would question if it should be more successfully leaning to the Leviathan as a Navy ship, but perhaps they didn't have the production means yet. That, and the Frigate still exists, which also isn't as great as a Leviathan.

Reviewer 5: To keep the Far North concept in play, the possible route is that the Republic could've just sold their old ships to Betelgeuse.  That happens a lot with surplus military equipment; sometimes you get a country developing something unique and fabulous, but in times of need, they happily accept outdated equipment to reinforce whatever their armies need.  We see that today IRL with Ukraine for instance.  Given the proximity of Betelgeuse to the Far North, it's feasible that in a time of desperation, related to piracy or economics, they may have reached out to the Republic to take those defunct Monitors off their hands, did a rapid refit and restoration, used them to engage pirates until they had enough newer-model Firebirds?  The flip side is that the Monitors ended up sold off to the public, and ultimately in the hands of pirates, anarchists and a few merchants here and there.

Reviewer 4: Adding the Republic motto is a little cheeky, I have to admit, but I find it funny 😁 
```suggestion
	description `In the early days of the Republic, planetary governments and corporations often fielded their own fleets; only the Deep had any form of a combined navy. The Republic's rapid expansion caused disruption throughout human space, and Earth often found itself directing fleets of loyal planets to fight against defiant states or pirate incursions. With a need to bring "peace among the stars," as the Republic motto goes, the newly formed Republic Navy commissioned the shipyards on Luna to design a robust warship that would be able to project the power of a united Republic. The resulting ship, the Monitor, served as the Navy's primary warship until the introduction of the more advanced Frigate.`
	description `	Once the backbone of the early Navy, the aging Monitors that survive today have found themselves in the hands of those against whom they once fought, including groups of anarchists that now populate the north of human space.`
```

Author: Should there be any mention of influence by the Firebird in the description? The two curved wings on the design now are taken from the Firebird/Leviathan, which give it Betelgeuse-esq looks, and it's true the Monitor is, at least now, a Northern ship. So perhaps something about it being influenced by the robust Firebird? If so, it would be a pretty wordy description. Or perhaps it's not necessary?

Reviewer 4: I think that this idea could be expressed in a mission of some sort; the proposed description is pretty wordy already, and I'm having trouble trimming it.

Reviewer 4: Okay I think I'm happy with this.
```suggestion
	description `When the Republic was first formed, planetary governments and corporations often fielded their own fleets; only the Deep had any form of a combined navy. The Republic's rapid expansion created both opportunity and disruption throughout human space, and Earth often found itself directing the fleets of its planets to fight pirates or defiant planets. With a need to bring "peace among the stars," as the Republic motto goes, the newly formed Republic Navy commissioned the shipyards on Luna to design a robust warship that would be able to project the power of a united Republic. The resulting ship, the Monitor, served as the mainstay of the Navy fleets until the introduction of the more advanced Frigate.`
	description `	The few Monitors that survive today have found themselves in the hands of those against whom they once fought, including groups of anarchists that now populate the north of human space.`
```

Reviewer 5: It's really hard to come up with a way to shave it down to a point where it's concise, without removing a bulk of the backstory lore.  The problem with overly long ship descriptions is that they can get in the way of functional play; I have to scroll through them to get to the ship stats so I can compare things.

I think I found a way to do it, without really altering the 'gist' of your work or redoing anything, you might like it actually.

```suggestion
	description `When the Republic first formed, planetary governments and corporations often fielded their own fleets; only the Deep had any form of a combined navy. Earth often found itself inefficiently directing those fleets to fight against pirates or defiant worlds. With a need to bring "peace among the stars," per the Republic motto, the newly formed Republic Navy commissioned the shipyards on Luna to design a robust warship that would be able to project the power of a united Republic. The resulting ship, the Monitor, served as the mainstay of the Navy fleets until the introduction of the more advanced Frigate.`
	description `The few Monitors that still fly today are remnants of those sold by the Navy to the government of Betelgeuse, before they joined the Republic. Of the Monitors that made it into private hands, some eventually found themselves passing through the black markets of the Far North.`
```

Now, I left out the _"The Republic's rapid expansion created both growth and disruption throughout human space, and Earth often found itself inefficiently directing its planetary fleets to fight against pirates or defiant worlds."_ because it's already implied in the game through other lore the player is already exploring; namely, the progression and facts involving the Free Worlds.  The redundancy, for a ship description, just doesn't seem necessary.

There should be more opportunities within the Anarchists to explore logic paths like that.

> 	description "	In a twisted act of irony, aging Monitors have eventually found themselves in the hands of those against whom they once fought, including groups of anarchists that now populate the north of human space."

Reviewer 3: ```suggestion
	description "	In a twisted act of irony, aging Monitors have found themselves in the hands of those against whom they once fought, including groups of anarchists that now populate the north of human space."
```

ship "Monitor"
	sprite "ship/monitor"
	thumbnail "thumbnail/monitor"
	attributes
		category "Medium Warship"
		"cost" 2900000
		"shields" 4900
		"hull" 2400
		"required crew" 20
		"bunks" 32
		"mass" 690
		"drag" 8.4
		"heat dissipation" .32
		"fuel capacity" 300
		"cargo space" 20
		"outfit space" 330
		"weapon capacity" 130
		"engine capacity" 90
		weapon
			"blast radius" 73
			"shield damage" 730
			"hull damage" 365
			"hit force" 1095
	outfits
		"Heavy Blaster" 3
		"Laser Turret" 2
		"Anti-Missile Turret"
		
		"KP-6 Photovoltaic Array" 3
		"nGVF-BB Fuel Cell"
		"LP036a Battery Pack"
		"D23-QP Shield Generator"
		"Laser Rifle" 6
		"Fragmentation Grenades" 4
		
		"X3700 Ion Thruster"
		"X3200 Ion Steering"
		"Hyperdrive"
		
	engine -20.5 92
	engine 20.5 92
	gun 0 -94.5 "Heavy Blaster"
	gun -11 -86 "Heavy Blaster"
	gun 11 -86 "Heavy Blaster"
	turret -17.5 -19.5 "Laser Turret"
	turret 17.5 -19.5 "Laser Turret"
	turret 0 17 "Anti-Missile Turret"
	leak "leak" 30 40
	leak "flame" 40 80
	leak "big leak" 70 30
	explode "tiny explosion" 20
	explode "small explosion" 40
	explode "medium explosion" 20
	explode "large explosion" 5
	"final explode" "final explosion medium"

> 	description "Monitor."

Reviewer 3: ```suggestion
	description "Before the Navy was founded, the North suffered from anarchist attacks and piracy. An enterprising young engineer founded Betelguese Shipyards to create the Monitor as a way of protecting against the criminals in the upper regions of human space."
```

Author: I think using the Enforcer's description is a good reference, as it too is a pre-Navy-era security ship, albeit more for the fringes:

> Prior to the formation of the galactic Republic, many star systems were owned by "governing corporate entities," with the vast majority of space being inaccessible to private citizens. The only exception to this rule were the pirate bands, whose few ships were often stolen or rebellious corporate fleets of their own, yet who had an outsized impact on interstellar trade. The necessity for corporations to protect their operations meant that Enforcer was one of the most common vessels used for security and defense. The ship boasts an armament of four gun ports and a turreted mount, all packed on a small but defensible frame that was powerful for its time.```

That said, the Monitor would be less of a sort of "contracted security ship" for a given corporation, and more of a powerhouse enforcer for more core worlds or valued areas where Piracy was a problem as opposed to simply being due to the distant location/underdevelopment. Kind of what Cruisers do now. So perhaps the following:

> 	description "Monitor."

Author: ```suggestion
	description "Before the time of the Navy, the formation of the Republic caused civil unrest amongst the inhabitants of several worlds. While some corporations happily joined the power, others wished to stay independent or even rebel against it. To both quell the uncertainty and bring order to the growing Republic, the Monitor was manufactured as a powerful security ship, armed with three large forward guns and three supportive turret mounts. These ships would also see use in the Alpha Wars, although they were eventually made obsolete by more robust designs such as the Firebird."
description "	In a twisted act of irony, the aging Monitor would eventually find itself in the hands of those who it once protected the Republic against, amongst groups of anarchists that now populate the north of human space."
```

Reviewer 3: Yep, that's better. You're missing the tab at the beginning of the second paragraph, thoug.

ship "Monitor"
	sprite "ship/monitor"
	thumbnail "thumbnail/monitor"
	attributes
		category "Medium Warship"
		"cost" 2900000
		"shields" 4900
		"hull" 2400
		"required crew" 20
		"bunks" 32
		"mass" 690
		"drag" 8.4
		"heat dissipation" .32
		"fuel capacity" 300
		"cargo space" 20
		"outfit space" 330
		"weapon capacity" 130
		"engine capacity" 90
		weapon
			"blast radius" 73
			"shield damage" 730
			"hull damage" 365
			"hit force" 1095
	outfits
		"Heavy Blaster" 3
		"Laser Turret" 2
		"Anti-Missile Turret"
		
		"KP-6 Photovoltaic Array" 3
		"nGVF-BB Fuel Cell"
		"LP036a Battery Pack"
		"D23-QP Shield Generator"
		"Laser Rifle" 6
		"Fragmentation Grenades" 4
		
		"X3700 Ion Thruster"
		"X3200 Ion Steering"
		"Hyperdrive"
		
	engine -20.5 92
	engine 20.5 92
	gun 0 -94.5 "Heavy Blaster"
	gun -11 -86 "Heavy Blaster"
	gun 11 -86 "Heavy Blaster"
	turret -17.5 -19.5 "Laser Turret"
	turret 17.5 -19.5 "Laser Turret"
	turret 0 17 "Anti-Missile Turret"
	leak "leak" 30 40
	leak "flame" 40 80
	leak "big leak" 70 30
	explode "tiny explosion" 20
	explode "small explosion" 40
	explode "medium explosion" 20
	explode "large explosion" 5
	"final explode" "final explosion medium"

> 	description "Before the time of the Navy, the formation of the Republic caused civil unrest amongst the inhabitants of several worlds. While some corporations happily joined the power, others wished to stay independent or even rebel against governance. To both quell the uncertainty and bring order to the growing Republic, the Monitor was manufactured as a powerful security ship, armed with three large forward guns and three supportive turret mounts. These ships would also see use in the Alpha Wars, although they were eventually made obsolete by more robust designs such as the Firebird."

Reviewer 6: A couple of minor changes that I think make the language stronger.
```suggestion
	description "Before the time of the Navy, the formation of the Republic caused civil unrest amongst the inhabitants of several worlds. While some corporations happily joined the fledgling power, others wished to stay independent or even rebelled against governance. To both quell this uncertainty and bring order to the growing Republic, the Monitor was manufactured as a powerful security ship, armed with three large forward guns and three supportive turret mounts. These ships would also see use in the Alpha Wars, although they were eventually made obsolete by more robust designs such as the Firebird."
```

> 	description "	In a twisted act of irony, aging Monitors would eventually find itself in the hands of those who it once protected the Republic against, amongst groups of anarchists that now populate the north of human space."

Reviewer 6: I think this sentence structure sounds more natural.
```suggestion
	description "	In a twisted act of irony, aging Monitors have eventually found themselves in the hands of those against whom they once fought, including groups of anarchists that now populate the north of human space."
```
